lib/artifactory.rb

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 
+require 'time'
```

Comment:
We should place this require in the affected lower level file. My guess is that file is `lib/artifactory/resources/artifact.rb`.